app/models/user.rb

```diff
@@ -0,0 +1,19 @@
+class User < ActiveRecord::Base
```

Comment:
@stephmarx @rfleury2 let's take some time to check up on our editor settings and our personal habits around using spaces v tabs when we're working on our code, Github is showing our tabs all weird!